content/src/metrics.ts

```diff
@@ -85,5 +85,10 @@ export const metricsDeclaration = validateMetricsDeclaration({
     help: 'Pending downloading jobs',
     type: 'gauge',
     labelNames: ['entity_type']
+  },
+  dcl_files_migrated: {
+    help: 'Files migrated to new folder structure',
+    type: 'counter',
+    labelNames: []
   }
```

Comment:
This metric will not be useful since it doesn't change during the lifecycle of the program. A log will suffice